<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.40">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Jonathan F. Jupke">

<title>test</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="00_index_files/libs/clipboard/clipboard.min.js"></script>
<script src="00_index_files/libs/quarto-html/quarto.js"></script>
<script src="00_index_files/libs/quarto-html/popper.min.js"></script>
<script src="00_index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="00_index_files/libs/quarto-html/anchor.min.js"></script>
<link href="00_index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="00_index_files/libs/quarto-html/quarto-syntax-highlighting-549806ee2085284f45b00abea8c6df48.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="00_index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="00_index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="00_index_files/libs/bootstrap/bootstrap-8a79a254b8e706d3c925cde0a310d4f0.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<link href="00_index_files/libs/tabwid-1.1.3/tabwid.css" rel="stylesheet">

<script src="00_index_files/libs/tabwid-1.1.3/tabwid.js"></script>


  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">test</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Jonathan F. Jupke </p>
          </div>
  </div>
    
  
    
  </div>
  


</header>


<section id="introduction" class="level1">
<h1>Introduction</h1>
<p>A tool in the management and research of ecosystems is to divide them into different types: river types, ecoregions, habitat types. These examples serve to highlight the breadth of systems, spatial scales, and methods employed in this exercise. These classifications can be usefull for several discinct reasons. <!-- Why are they useful --> They are a easy to use representation of the environmental context and might be considered as a latent, discrete gradient capturing the relevant environmental variables like their continuous counterpart, the ordination axis. This environmental context can then be used to calibrate indicator metrics in biomonitoring <span class="citation" data-cites="szoszkiewiczMacrophyteIndexRivers2019">(<a href="#ref-szoszkiewiczMacrophyteIndexRivers2019" role="doc-biblioref">Szoszkiewicz et al. 2019</a>)</span>. This is necessary as responses of biota to natural gradients <span class="citation" data-cites="bogota-gregoryGeomorphologicalHabitatType2023 alahuhtaComparativeAnalysisReveals2017">(<a href="#ref-alahuhtaComparativeAnalysisReveals2017" role="doc-biblioref">Alahuhta et al. 2017</a>; <a href="#ref-bogota-gregoryGeomorphologicalHabitatType2023" role="doc-biblioref">Bogotá-Gregory et al. 2023</a>)</span> and anthropogenic stressors <span class="citation" data-cites="denisonIntegratingRegionalFrameworks2021 bevanRegionalBiomesOutperform2024 pharaohPotentialDriversChanging2024">(<a href="#ref-denisonIntegratingRegionalFrameworks2021" role="doc-biblioref">Denison et al. 2021</a>; <a href="#ref-pharaohPotentialDriversChanging2024" role="doc-biblioref">Pharaoh et al. 2024</a>; <a href="#ref-bevanRegionalBiomesOutperform2024" role="doc-biblioref">Bevan et al. 2024</a>)</span> can differ between ecosystem types. Such context or region-dependence has also been observed for trait-environment relationsships <span class="citation" data-cites="yangWhyFunctionalTraits2018 burnerTraitsMediateNiches2021 koppJointSpeciesDistribution2023">(<a href="#ref-yangWhyFunctionalTraits2018" role="doc-biblioref">Yang et al. 2018</a>; <a href="#ref-burnerTraitsMediateNiches2021" role="doc-biblioref">Burner et al. 2021</a>; <a href="#ref-koppJointSpeciesDistribution2023" role="doc-biblioref">Kopp et al. 2023</a>)</span>. They are also used as a controlling variable when planning experiments. When we try to minimize the impact certain environmental features, we can focus our sampling to a small selection of types (ref). We often hold different protection goals for different regions of habitats - a typology can then facilitating the selection, combination, or priorisation of management actions. Lastly, they can also serve as a proxy for species ranges, where detailed information is lacking <span class="citation" data-cites="pinkertPotentialEcoregionalRange2023">(<a href="#ref-pinkertPotentialEcoregionalRange2023" role="doc-biblioref">Pinkert et al. 2023</a>)</span>.</p>
<!-- Assumption -->
<p>Typologies are models. Rather than describing nature, they superimpose their simplified view. A view that can be usefull if they meet the assumption that ecological communities within types are more similar to oneanother than to communities from other types. Generally this property is known as external cluster validity (insert short description), and in this context more specifically as biological validity (ref). With the advent of typology systems in different management schemes (Clean Water act, Water Framework Directive, more examples - also check CWA again.) interest in biovalidation grew and noumerous systems have to date been evaluated against noumerous (new word) taxonomic groups.</p>
<p>Provide some examples.</p>
<!-- Issues with studies up to this point -->
<p>The available studies on the biovalidity of typology systems suffer from two flaws. First, there is contention which method should be used to determine biovalidity. Different studies use different methods, which reduces comparability among them. It remians difficult to compare the biovalidity of typology systems across taxa and places, because we lack an empirically grounded notion of how biovalid a biovalid typology systems is. In lieu, two practices have taken hold. Biovalidity is judged against unrealistic heurisitcs without ecological foundations or against statistical significane. Effect size heuristics exist for some commonly used metrics but not all. <!-- provide examples --> They are not specifically tailored to evaluate biovalidity. They are based on practically irrelevant considerations. <!-- Provide example here --> The fact that these heuristics are disconnected from ecological realities is evident as they are context-, scale-, and taxon-independent. All three factors should influence the result. <!-- refereneces --></p>
<p>Dichotomous decisions based on <em>p</em>-values suffer from a slew of issues. Here, we wish to highlight X issues relevent whithin this context. P-value depend on effect size and sample size. Large scale ecological studies typically have a large number of sampling sites, thus rendering even miniscule effect sizes statistcally significant. <!-- more issues pertinent here? --> <!-- more references here --> A move toward <em>a priori</em> determined relevant effect sizes seems frutiful <span class="citation" data-cites="popovicFourPrinciplesImproved2024">(<a href="#ref-popovicFourPrinciplesImproved2024" role="doc-biblioref">Popovic et al. 2024</a>)</span>. Other ways to adress these issues have been proposed but will be considered here <span class="citation" data-cites="crouchFalseEvidenceRate2025 maierJustifyYourAlpha2022 wulffHowWhyAlpha2023">(<a href="#ref-maierJustifyYourAlpha2022" role="doc-biblioref">Maier and Lakens 2022</a>; <a href="#ref-wulffHowWhyAlpha2023" role="doc-biblioref">Wulff and Taylor 2023</a>; <a href="#ref-crouchFalseEvidenceRate2025" role="doc-biblioref">Crouch 2025</a>)</span></p>
<!-- We aim to -->
<p>We aim to provide a comprehensive set of benchmarks for different biovalidity metrics. These benchmarks will cover different taxonomic groups and different spatial scales. We will train a large number regression models on independent empirical data sets and use these models to simulate new biological data. These simulations will be based on artificial typology systems along a gradient of low to high expected biovalidity, thus enabling us to map plausible ranges for different metrics, spatial scales, and taxa.</p>
<section id="methods" class="level2">
<h2 class="anchored" data-anchor-id="methods">Methods</h2>
<section id="the-data-sets" class="level3">
<h3 class="anchored" data-anchor-id="the-data-sets">The data sets</h3>
<!-- provenance of data -->
<p>We have compiled observational data sets of diatoms, macroinvertebrate, fish, and aquatic macropyhtes that are freely available plus additional ones from our collaborators. An overview over all individual data sets is provided in in supplementary materials.<br>
These data sets span vastly different temporal and spatial ranges (add concrete numbers). Based on these data sets we will determine plausible ranges for the importance of comunity assembly processes and simulate data with varying degrees of adherence to underlying typology sytems. We will work on each data set seperately, as differences between data sets might confound ecologically relevant signals.</p>
<!-- subsampling -->
<p>Each data set will be subsampled. Within each data set, we will stratify the subsampling by year. Within each year, we will restrict it to the three consective months with the most samples. If the number of samples in these months (<span class="math inline">\(N_{months}\)</span>) &lt; 100 samples, this year will not be evaluated for this data set. If <span class="math inline">\(N_{months} \ge 100\)</span> samples, our subsampling methodology will incorporate a descending-frequency approach based on one hundred-sample increments. The process will begin by dividing <span class="math inline">\(N_{months}\)</span> by 100 (rounded down) to determine the number of increments <span class="math inline">\(N_{inc}\)</span>. For each increment <span class="math inline">\(i = 1, \dots, N_{inc}\)</span>, we will generate <span class="math inline">\((N_{inc} - (i-1))\)</span> random subsamples, each containing <span class="math inline">\(i \times 100\)</span> samples. When the remainder from dividing <span class="math inline">\(N_{months}\)</span> by 100 exceeds 0.5, we will include one additional sample at the maximum size. This approach ensures thorough sampling coverage while maintaining computational feasibility. For each subsample we determined the minimum, mean, median, and maximum spatial distance between samples with the R package sf (ref). We then remove taxa that occur in less than 5% of sampling sites.</p>
</section>
<section id="environmental-data" class="level3">
<h3 class="anchored" data-anchor-id="environmental-data">Environmental data</h3>
<p>We will compile and extensive data base of environmental variables relevant for freshwater organisms (Table 1). Each variable will be summarized at the catchment scale, where catchments are derived from the EU hydro DEM catchment data base (REF).</p>
<div class="cell">
<div class="cell-output-display">
<div class="tabwid"><style>.cl-ff08cd68{table-layout:auto;}.cl-fefcadda{font-family:'Arial';font-size:11pt;font-weight:bold;font-style:normal;text-decoration:none;color:rgba(0, 0, 0, 1.00);background-color:transparent;}.cl-fefcade4{font-family:'Arial';font-size:10pt;font-weight:normal;font-style:normal;text-decoration:none;color:rgba(0, 0, 0, 1.00);background-color:transparent;}.cl-ff010efc{margin:0;text-align:left;border-bottom: 0 solid rgba(0, 0, 0, 1.00);border-top: 0 solid rgba(0, 0, 0, 1.00);border-left: 0 solid rgba(0, 0, 0, 1.00);border-right: 0 solid rgba(0, 0, 0, 1.00);padding-bottom:4pt;padding-top:4pt;padding-left:4pt;padding-right:4pt;line-height: 1;background-color:transparent;}.cl-ff0138c8{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 1pt solid rgba(102, 102, 102, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138c9{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 1pt solid rgba(102, 102, 102, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138d2{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 1pt solid rgba(102, 102, 102, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138d3{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 1pt solid rgba(102, 102, 102, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138d4{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 0 solid rgba(0, 0, 0, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138dc{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 0 solid rgba(0, 0, 0, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138dd{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 0 solid rgba(0, 0, 0, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138e6{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 0 solid rgba(0, 0, 0, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138e7{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 1pt solid rgba(102, 102, 102, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138e8{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 1pt solid rgba(102, 102, 102, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138e9{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 1pt solid rgba(102, 102, 102, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138f0{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 1pt solid rgba(102, 102, 102, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138f1{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 1pt solid rgba(102, 102, 102, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138f2{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 1pt solid rgba(102, 102, 102, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138fa{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 1pt solid rgba(102, 102, 102, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}.cl-ff0138fb{background-color:transparent;vertical-align: middle;border-bottom: 1pt solid rgba(102, 102, 102, 1.00);border-top: 1pt solid rgba(102, 102, 102, 1.00);border-left: 1pt solid rgba(102, 102, 102, 1.00);border-right: 1pt solid rgba(102, 102, 102, 1.00);margin-bottom:0;margin-top:0;margin-left:0;margin-right:0;}</style><table data-quarto-disable-processing="true" class="cl-ff08cd68"><thead><tr style="overflow-wrap:break-word;"><th class="cl-ff0138c8"><p class="cl-ff010efc"><span class="cl-fefcadda">Variable</span></p></th><th class="cl-ff0138c9"><p class="cl-ff010efc"><span class="cl-fefcadda">Spatial Scale</span></p></th><th class="cl-ff0138d2"><p class="cl-ff010efc"><span class="cl-fefcadda">Data product</span></p></th><th class="cl-ff0138d3"><p class="cl-ff010efc"><span class="cl-fefcadda">Reference</span></p></th></tr></thead><tbody><tr style="overflow-wrap:break-word;"><td class="cl-ff0138d4"><p class="cl-ff010efc"><span class="cl-fefcade4">Rainfall erosivity</span></p></td><td class="cl-ff0138dc"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138dd"><p class="cl-ff010efc"><span class="cl-fefcade4">GloREDa</span></p></td><td class="cl-ff0138e6"><p class="cl-ff010efc"><span class="cl-fefcade4">Panagos et al. (2023)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138e7"><p class="cl-ff010efc"><span class="cl-fefcade4">Soil Organic Carbon</span></p></td><td class="cl-ff0138e8"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138e9"><p class="cl-ff010efc"><span class="cl-fefcade4">European Soil Database v2.0</span></p></td><td class="cl-ff0138f0"><p class="cl-ff010efc"><span class="cl-fefcade4">Panagos et al. (2022)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Soil pH in Water</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">EcoDataCube</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Witjes et al. (2023)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Slope</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">Hydrography90m</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Amatulli et al. (2022)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Roughness</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">Geomorpho90m</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Amatulli et al. (2020)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Floodplain Area</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">Potential Flood prone Area</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">EEA (2020)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Mean annual discharge</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">Copernicus C3S</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Berg et al. (2021)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Minimum annual discharge</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">Copernicus C3S</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Berg et al. (2021)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Saturated Soil Water Content</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">Soil Hydraulic Properties</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Tóth et al. (2015)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Upstream catchment area</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">HydroATLAS</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Linke et al (2019)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Maximum annual discharge</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">Copernicus C3S</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Berg et al. (2021)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Snow Cover</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">HydroATLAS</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Linke et al (2019)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Segment Sinuosity</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">EU Hydro</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">EEA (2019)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Valley Bottom Flatness Index</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">EcoDataCube</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Witjes et al. (2023)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Stream Power Index</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">Hydrography90m</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Amatulli et al. (2022)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Lake Index</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">HydroLAKES</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Messager et al. (2016); Snelder et al. (2005)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Elevation</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Region, Ecosystem</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">Hydrography90m</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Amatulli et al. (2022)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Groundwater Table Depth</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Region</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">HydroATLAS</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Linke et al. (2019)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Glaciated Area in Catchment</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Region</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">Randolph Glacier Inventory</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">RGI Consortium (2017)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Precipitation of Wettest Month</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Region</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">CHELSA-BIOCLIM+</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Brun et al. (2022)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Precipitation of Driest Month</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Region</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">CHELSA-BIOCLIM+</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Brun et al. (2022)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Minimum Temperature Coldest Month</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Region</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">CHELSA-BIOCLIM+</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Brun et al. (2022)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Maximum Temperature Warmest Month</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Region</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">CHELSA-BIOCLIM+</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Brun et al. (2022)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Bedrock Geology</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Region</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">IHME</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Cornu et al. (2013)</span></p></td></tr><tr style="overflow-wrap:break-word;"><td class="cl-ff0138f1"><p class="cl-ff010efc"><span class="cl-fefcade4">Catchment soil type</span></p></td><td class="cl-ff0138f2"><p class="cl-ff010efc"><span class="cl-fefcade4">Region</span></p></td><td class="cl-ff0138fa"><p class="cl-ff010efc"><span class="cl-fefcade4">European Soil Database v2.0</span></p></td><td class="cl-ff0138fb"><p class="cl-ff010efc"><span class="cl-fefcade4">Panagos et al. (2022)</span></p></td></tr></tbody></table></div>
</div>
</div>
<!-- | Variable | Spatial Scale | Data product | Reference |
|----------|--------------|--------------|-----------|
| Rainfall erosivity | Ecosystem | GloREDa | Panagos et al. (2023) |
| Soil Organic Carbon | Ecosystem | European Soil Database v2.0 | Panagos et al. (2022) |
| Soil pH in Water | Ecosystem | EcoDataCube | Witjes et al. (2023) |
| Slope | Ecosystem | Hydrography90m | Amatulli et al. (2022) |
| Roughness | Ecosystem | Geomorpho90m | Amatulli et al. (2020) |
| Floodplain Area | Ecosystem | Potential Flood prone Area | EEA (2020) |
| Mean annual discharge | Ecosystem | Copernicus C3S | Berg et al. (2021) |
| Minimum annual discharge | Ecosystem | Copernicus C3S | Berg et al. (2021) |
| Saturated Soil Water Content | Ecosystem | Soil Hydraulic Properties | Tóth et al. (2015) |
| Upstream catchment area | Ecosystem | HydroATLAS | Linke et al (2019) |
| Maximum annual discharge | Ecosystem | Copernicus C3S | Berg et al. (2021) |
| Snow Cover | Ecosystem | HydroATLAS | Linke et al (2019) |
| Segment Sinuosity | Ecosystem | EU Hydro | EEA (2019) |
| Valley Bottom Flatness Index | Ecosystem | EcoDataCube | Witjes et al. (2023) |
| Stream Power Index | Ecosystem | Hydrography90m | Amatulli et al. (2022) |
| Lake Index | Ecosystem | HydroLAKES | Messager et al. (2016); Snelder et al. (2005) |
| Elevation | Region, Ecosystem | Hydrography90m | Amatulli et al. (2022) |
| Groundwater Table Depth | Region | HydroATLAS | Linke et al. (2019) |
| Glaciated Area in Catchment | Region | Randolph Glacier Inventory | RGI Consortium (2017) |
| Precipitation of Wettest Month | Region | CHELSA-BIOCLIM+ | Brun et al. (2022) |
| Precipitation of Driest Month | Region | CHELSA-BIOCLIM+ | Brun et al. (2022) |
| Minimum Temperature Coldest Month | Region | CHELSA-BIOCLIM+ | Brun et al. (2022) |
| Maximum Temperature Warmest Month | Region | CHELSA-BIOCLIM+ | Brun et al. (2022) |
| Bedrock Geology | Region | IHME | Cornu et al. (2013) |
| Catchment soil type | Region | European Soil Database v2.0 | Panagos et al. (2022) | -->
<!-- Space -->
<p>We will capture the impact of space using one of two different approaches: Moran’s Eigenvector Maps (MEM, ref) or Asymmetric Eigenvector Maps (AEM, ref). (ref) MEM will be used of the largest spatial distance within a subsample exceeds T, otherwise we will use AEM. For both we will evaluate the statistical significance of Moran’s I of the eigenvectos with permutation tests <span class="citation" data-cites="cliffSpatialAutocorrelation1973">(<a href="#ref-cliffSpatialAutocorrelation1973" role="doc-biblioref">Cliff and Ord 1973</a>)</span>, and only keept those eigenvectors which had a statistically significant Moran’s I value. Both will be implemented with the adespatial R package version … (ref).</p>
</section>
<section id="modellings-species-communities" class="level3">
<h3 class="anchored" data-anchor-id="modellings-species-communities">Modellings species communities</h3>
<!-- describe HMSC setup -->
<p>To establish the relationship between the presence/ absence of organisms and environmental variables, we will use hierarchical modelling of species communities (HMSC, ref) from the corresponding R package Hmsc (ref). This bayesian hierarchical latent variable model allows the incorporation of phylogeny and traits and has been favoribly compared to similar methdods (ref).<br>
We will include a non-spatiotemporal random factor for each sample. The model will be fit with a probit residual distribution. <!-- TODO --> Model fit will be evaluated with the potential scale reduction factor and choose a second statistic. By default, we will run models with two chains, 2000 samples, a thinning of 2, and a transient period of 2000 samples. If either of potential scale reduction factor or Welch’s statistic suggest that the chains have not converged or that the posterior is not stationary for &gt; 10% of regression parameters, we will iteratively increase the number of samples up to 10000. Thinning will be increased in parallel, so that it is awlways the number of chains devided by 1000, and the length of the transient equals the number of samples.</p>
<!-- Traits and phenology where available -->
<p>Where available, we include both functional traits and phylogenetic data in the HMSC model. For fish, funcitonal data is available through Traits of Freshwater Fish <span class="citation" data-cites="lecocqTOFFDatabaseTraits2019">(TOFF, <a href="#ref-lecocqTOFFDatabaseTraits2019" role="doc-biblioref">Lecocq et al. 2019</a>)</span> and the phylogenetic data from <span class="citation" data-cites="raboskyInverseLatitudinalGradient2018">Rabosky et al. (<a href="#ref-raboskyInverseLatitudinalGradient2018" role="doc-biblioref">2018</a>)</span>. For diatoms, … . For macroinvertebrates, functional data is available through <span class="citation" data-cites="kunzTacklingInconsistenciesFreshwater2022">Kunz et al. (<a href="#ref-kunzTacklingInconsistenciesFreshwater2022" role="doc-biblioref">2022</a>)</span>, but highly resolved phylogenetic data is only available for certain orders <span class="citation" data-cites="garcia-gironTimecalibratedTreeLife2024">(see <a href="#ref-garcia-gironTimecalibratedTreeLife2024" role="doc-biblioref">García-Girón et al. 2024</a>)</span>. For macrophytes,</p>
<!-- describe variation partioning -->
<p>Rather than evaluating the explained variation based on point estimates derived from the parameter’s posterior distribution, we will compute the full posterior of the VP (see ref) and use point estiamtes from the latter. Due to Jensen’s inequality these two values are not equal.</p>
<p>We will interpret all the variance explained by environmental variables as abiotic filering, the variance explained by spatial eigenvectors as spatial filtering, and the variance of the latent variables as biotic interactions. The last point is highly contentions <span class="citation" data-cites="blanchetCooccurrenceNotEvidence2020 dormannBioticInteractionsSpecies2018 valleSpeciesAssociationsJoint2023">(<a href="#ref-dormannBioticInteractionsSpecies2018" role="doc-biblioref">Dormann et al. 2018</a>; e.g., <a href="#ref-blanchetCooccurrenceNotEvidence2020" role="doc-biblioref">Blanchet et al. 2020</a>; <a href="#ref-valleSpeciesAssociationsJoint2023" role="doc-biblioref">Vallé et al. 2023</a>)</span>, as the latent variables can also captured unmeasured enviornmental driveres. In our case this will likely include local scale variables such as sediment composition. We expect therefore to slightly overestimate the relevance of biotic interactions and will keep this in mind in any interpretations of the results. Tjur’s <span class="math inline">\(R^2\)</span> <span class="citation" data-cites="tjurCoefficientsDeterminationLogistic2009">(<a href="#ref-tjurCoefficientsDeterminationLogistic2009" role="doc-biblioref">Tjur 2009</a>)</span> will be used to evaluate the overall explained variation in the biotic data.<br>
The unexplained variance <span class="math inline">\(1-R^2_{Tjur}\)</span>, will be interpreted as stoachistcity in communitiy composition. Each of these values will be averaged across species. For each fitted model, we thus obtain a vector of four numbers, which sum to 1, which estimate the assembly mechanisms of this metacommunity. Over all the models we will run, we will be able to derive a multivariate proability distribution of assembly mechanisms. As each mechanism itself can range between 0 and 1, their distribution can likely be captured by a beta distribution. The Dirichlet distribution is a multivariate distribution with beta marginals, and thus able to represent this.</p>
</section>
<section id="simulating-typologies" class="level3">
<h3 class="anchored" data-anchor-id="simulating-typologies">Simulating typologies</h3>
<p>The fitted models will be used to simulate new bitoic data.</p>
<!-- non fuzzy -->
<!-- fuzzy     -->
</section>
<section id="evaluating-typologies" class="level3">
<h3 class="anchored" data-anchor-id="evaluating-typologies">Evaluating typologies</h3>
<p>We will evaluate each typology with a selection of recommended or practically used methods. All methods are shortly introduced below. Consider a set of <span class="math inline">\(N\)</span> sites classified into <span class="math inline">\(T\)</span> distinct types <span class="math inline">\(t_1, t_2, \dots, t_T \in \tau\)</span>, where <span class="math inline">\(\tau\)</span> represents the complete typology system. Let <span class="math inline">\(x_i\)</span> be a site and <span class="math inline">\(\tau(x_i)\)</span> the function that returns its type assignment. For any site <span class="math inline">\(x_i\)</span>, let <span class="math inline">\(\bar{d}(x_i,\mathbf{x}_{\tau(x_i)})\)</span> represent the mean dissimilarity between <span class="math inline">\(x_i\)</span> and all other sites within its assigned type <span class="math inline">\(\tau(x_i)\)</span>. For any other type <span class="math inline">\(t_x \in \tau \setminus \{\tau(x_i)\}\)</span>, let <span class="math inline">\(\bar{d}(x_i,\mathbf{x}_{t_x})\)</span> denote the mean dissimilarity between <span class="math inline">\(x_i\)</span> and all sites in type <span class="math inline">\(t_x\)</span>. The neighbor type of <span class="math inline">\(x_i\)</span>, denoted as <span class="math inline">\(b(x_i)\)</span>, is defined as the type (excluding <span class="math inline">\(\tau(x_i)\)</span>) that minimizes the mean dissimilarity: <span class="math display">\[b(x_i)=\underset{x \in \tau \setminus \{\tau(x_i)\}}{argmin}\ \bar{d}(x_i,\mathbf{s}_{t_x})\]</span> The silhouette width quantifies how well <span class="math inline">\(x_i\)</span> fits within its assigned type compared to its neighbor type. For site <span class="math inline">\(x_i\)</span>, it is computed as:<br>
<span class="math display">\[sw(x_i) = \dfrac{\bar{d}(x_i, \mathbf{x}_{b(x_i)}) - \bar{d}(x_i,\mathbf{x}_{\tau(x_i)})}{max\big\{\bar{d}(x_i, \mathbf{x}_{b(x_i)}), \bar{d}(x_i,\mathbf{x}_{t_{\tau(x_i)}})\big\}}\]</span></p>
<p>The Average Silhouette Width (ASW) is calculated as the mean of individual silhouette widths across all sites: <span class="math display">\[ASW = \dfrac{1}{N}\Sigma_{i=1}^N sw(x_i)\]</span> The ASW ranges from -1 to 1, where -1 indicates the poorest possible type assignments and 1 represents optimal type assignments.</p>
<!-- Classification Strength -->
<p>The Classification Strength <span class="citation" data-cites="vansickleUsingMeanSimilarity1997">(CS, <a href="#ref-vansickleUsingMeanSimilarity1997" role="doc-biblioref">Van Sickle 1997</a>)</span> provides another measure to evaluate typology systems by comparing similarities within and among types. Given a similartiy matrix <span class="math inline">\(\mathbf{S}\)</span> with elements <span class="math inline">\(s_{ij}\)</span> where <span class="math inline">\(i\)</span> and <span class="math inline">\(j\)</span> index over sites, we define the within type similarity for type T as: <span class="math display">\[W_T = \dfrac{1}{K_T} \sum_{i = 1}^N \sum_{\substack{j = 1\\ j \neq i}}^N s_{ij} \times \epsilon_{ij}\]</span></p>
<p>where <span class="math inline">\(\epsilon_{ij} = 1\)</span> if sites <span class="math inline">\(i\)</span> and <span class="math inline">\(j\)</span> both belong to type <span class="math inline">\(T\)</span> (i.e., <span class="math inline">\(\tau(s_i)=\tau(s_j) = T\)</span>), and <span class="math inline">\(0\)</span> otherwise. <span class="math inline">\(K_T\)</span> represents the total number of pairwise comparisons within type <span class="math inline">\(T\)</span>, ensuring <span class="math inline">\(W_T\)</span> represents the mean similarity among sites of type <span class="math inline">\(T\)</span>. The among-type similarity for type <span class="math inline">\(T\)</span> is defined analogously: <span class="math display">\[B_T = \dfrac{1}{L_T} \sum_{i = 1}^N \sum_{\substack{j = 1\\ j \neq i}}^N s_{ij} \times (1-\epsilon_{ij})\]</span> where <span class="math inline">\(L_T\)</span> is the number of comparisons between sites in type <span class="math inline">\(T\)</span> and sites in all other types The type-specific classification strength is calculated as: <span class="math display">\[CS_t = W_t - B_t\]</span> This measure quantifies how much more similar sites are to others within their assigned type compared to sites in different types. The difference to SW is, that CS considers all other types and not only the neighbor. The Classification Strength for the complete typology system is computed as a weighted average of type-specific Classification Strengths: <span class="math display">\[CS = \sum_t^T \dfrac{n_t}{N} CS_t\]</span> where <span class="math inline">\(n_t\)</span> is the number of sites in type <span class="math inline">\(t\)</span> and <span class="math inline">\(N\)</span> is the total number of sites. Similar to the ASW, CS values range from -1 to 1, with higher values indicating more distinct and well-defined types.</p>
<!-- ANOSIM -->
<p>The Analysis of Similarities <span class="citation" data-cites="clarkeNonparametricMultivariateAnalyses1993">(ANOSIM, <a href="#ref-clarkeNonparametricMultivariateAnalyses1993" role="doc-biblioref">Clarke 1993</a>)</span> evaluates typology systems using ranked similarities rather than raw similarity values. Let <span class="math inline">\(\mathbf{S}_{rank}\)</span> be the matrix of ranked similarities derived from <span class="math inline">\(\mathbf{S}\)</span>, where lower ranks are assigned to higher similarities. For any pair of sites <span class="math inline">\((i,j)\)</span>, the element <span class="math inline">\(s^{rank}{ij}\)</span> represents the rank of the corresponding similarity <span class="math inline">\(s_{ij}\)</span>. We define <span class="math inline">\(\bar{r}_W\)</span> as the mean rank of within-type similarities and <span class="math inline">\(\bar{r}_B\)</span> as the mean rank of between-type similarities. The ANOSIM R statistic is computed as: <span class="math display">\[R = (\bar{r}_B - \bar{r}_W)/(M/2)\]</span> where <span class="math inline">\(M = N(N-1)/2\)</span> Like ASW and CS, the ANOSIM R statistic ranges from -1 to 1 A permutational <em>p</em>-value for R can be computed.</p>
<p>Permutational multivariate analysis of variances <span class="citation" data-cites="andersonNewMethodNonparametric2001">(PERMANOVA, <a href="#ref-andersonNewMethodNonparametric2001" role="doc-biblioref">Anderson 2001</a>)</span> evaluates typology systems by analyzing the variance in dissimilarities among sites. For a dissimilarity matrix <span class="math inline">\(\mathbf{D}\)</span> with elements <span class="math inline">\(d_{ij}\)</span>, we compute two sums of squares: The total sum of squares: <span class="math display">\[SS_{total} = \dfrac{1}{N}\sum_{i=1}^{N-1} \sum_{j=i+1}^{N} d_{ij}^2\]</span></p>
<p>The within-type sum of squares: <span class="math display">\[SS_{within} = \dfrac{1}{N}\sum{i=1}^{N-1} \sum{j=i+1}^{N} d_{ij}\times \epsilon_{ij}\]</span></p>
<p>The pseudo F-ratio is then computed as: <span class="math display">\[F = \dfrac{SS_{within} / (T-1)}{SS_{total} / (N-T)}\]</span> Larger F-ratios indicate greater separation between type centroids in multivariate space. The statistical significance of the F-ratio can be assessed through a permutation test.</p>
<!-- non geometric methods -->
<p>Non-geometric measures evaluate typology systems directly from abundance or occurrence data, without requiring similarity metrics. These include the indicator species analysis <span class="citation" data-cites="dufreneSpeciesAssemblagesIndicator1997">(IndVal, <a href="#ref-dufreneSpeciesAssemblagesIndicator1997" role="doc-biblioref">Dufrêne and Legendre 1997</a>)</span>, Indicator Species Analysis Minimizing Intermediate Constancies <span class="citation" data-cites="robertsLabdsvOrdinationMultivariate2023">(ISAMIC, <a href="#ref-robertsLabdsvOrdinationMultivariate2023" role="doc-biblioref">Roberts 2023</a>)</span>, and Area Under the ζ-diversity Decline Curve <span class="citation" data-cites="jupkeEuropeanRiverTypologies2023">(AUCζ, <a href="#ref-jupkeEuropeanRiverTypologies2023" role="doc-biblioref">Jupke et al. 2023</a>)</span>.<br>
<!--IndVal --> The indicator species analysis combines two components for each taxon <span class="math inline">\(h\)</span> in type <span class="math inline">\(t\)</span>: concentration (<span class="math inline">\(A_{ht}\)</span>) and relative frequency (<span class="math inline">\(F_{ht}\)</span>). A perfect indicator (IndVal = 1.0) occurs exclusively in one type (perfect concentration) and is present in all samples of that type (maximum frequency). For the group-equalized IndVal variant, which accounts for uneven sample sizes across types <span class="citation" data-cites="decaceresImprovingIndicatorSpecies2010">(<a href="#ref-decaceresImprovingIndicatorSpecies2010" role="doc-biblioref">De Cáceres et al. 2010</a>)</span>, we compute the concentration component as: <span class="math display">\[A^g_{ht} = \frac{a^g_t}{a^g}\]</span> where <span class="math inline">\(a^g_t = \frac{N}{k}\sum_{i \in t}(a_i/N_i)\)</span> is the group-equalized sum of abundances in type <span class="math inline">\(t\)</span>, and <span class="math inline">\(a^g = \frac{N}{k}\sum_{i \in K}(a_i/N_i)\)</span> is the total group-equalized abundance. The frequency component as: <span class="math display">\[B_{ht} = \frac{n_{ht}}{N_t}\]</span> where <span class="math inline">\(n_{ht}\)</span> is the number of occurrences of taxon <span class="math inline">\(h\)</span> in type <span class="math inline">\(t\)</span>, and <span class="math inline">\(N_t\)</span> is the total number of sites in type <span class="math inline">\(t\)</span>. The group-equalized IndVal for taxon <span class="math inline">\(h\)</span> in type <span class="math inline">\(t\)</span> is then: <span class="math display">\[IndVal^g_{ht} = \sqrt{A^g_{ht} \times B_{ht}} \times 100\]</span> For each species, only the highest IndVal is considered. <span class="math display">\[IndVal^g_h = \max_{t}(IndVal^g_{ht})\]</span> Statistical significance of each IndVal is assessed through a permutation test. Since multiple tests are performed (one for each taxon), the resulting p-values are adjusted using Holm’s step-down procedure to control for multiple comparisons <span class="citation" data-cites="westfallResamplingbasedMultipleTesting1993">(<a href="#ref-westfallResamplingbasedMultipleTesting1993" role="doc-biblioref">Westfall and Young 1993</a>)</span>. Two metrics evaluate the overall quality of the typology system: 1. The proportion of significant indicator taxa (at <span class="math inline">\(\alpha = 0.05\)</span>), denoted as <span class="math inline">\(N_{sig}\)</span>: <span class="math display">\[N_{sig}=\frac{1}{H}\sum_{h=1}{H}I(p_h&lt;0.05)\]</span> where <span class="math inline">\(I()\)</span> is the indicator function, <span class="math inline">\(H\)</span> the number of species and <span class="math inline">\(p_h\)</span> is the adjusted p-value for taxon <span class="math inline">\(h\)</span>. 2. The mean adjusted p-value across all taxon-type combinations: <span class="math display">\[\bar{p} = \frac{1}{H} \sum_{h=1}^H p_h\]</span></p>
<p>A superior typology system is characterized by a higher <span class="math inline">\(N_{sig}\)</span> and a lower <span class="math inline">\(\bar{p}\)</span>.</p>
<p>The Indicator Species Analysis Minimizing Intermediate Constancies (ISAMIC) evaluates how consistently species occur within types. For each taxon <span class="math inline">\(h\)</span> and type <span class="math inline">\(t\)</span>, we calculate: <span class="math display">\[ISAMIC = \frac{\sum_{h=1}^{H}\left\{\left(2\sum_{t=1}^{T}|F_{ht}-0.5|\right)/T\right\}}{N}\]</span> The metric quantifies the predictability of species occurrence patterns. Values close to 1 indicate that species tend to be either consistently present or consistently absent within types Values close to 0 indicate that species occurrences are unpredictable, with frequencies close to 0.5 Higher ISAMIC values suggest a more effective typology system.</p>
<p>The Area Under the Zeta diversity Curve <span class="citation" data-cites="jupkeEuropeanRiverTypologies2023">(AUCζ, <a href="#ref-jupkeEuropeanRiverTypologies2023" role="doc-biblioref">Jupke et al. 2023</a>)</span> quantifies the rate at which shared taxa decrease across an increasing number of sampling sites. For a set of sites, the zeta (ζ) diversity of degree <span class="math inline">\(q\)</span> represents the mean number of species shared across <span class="math inline">\(q\)</span> sites <span class="citation" data-cites="huiZetaDiversityConcept2014">(<a href="#ref-huiZetaDiversityConcept2014" role="doc-biblioref">Hui and McGeoch 2014</a>)</span> <span class="math display">\[\zeta_q = \mathbb{E}\left[\bigcap_{j=1}^q y_j\right]\]</span> where <span class="math inline">\(y_j\)</span> represents the set of species in site <span class="math inline">\(j\)</span>. For any typology system, we compute the ζ-diversity within-types <span class="math inline">\(\zeta^w_q\)</span> using only sites from the same type and the ζ-diversity between-types <span class="math inline">\(\zeta^b_q\)</span> using sites from different types. As ζ-diversity declines monotonically with increasing <span class="math inline">\(q\)</span>, we can compute the area under the decline curves to quantify the speed and magnitude of turnover. The AUCζ for is calculated as: <span class="math display">\[AUC\zeta^w = \frac{1}{\zeta_1}\sum_{q=2}^{10} \frac{\zeta^w_i + \zeta^w_{i-1}}{2}\]</span> <span class="math display">\[AUC\zeta^b = \frac{1}{\zeta_1}\sum_{q=2}^{10} \frac{\zeta^b_i + \zeta^b_{i-1}}{2}\]</span> We devide by <span class="math inline">\(zeta_1\)</span> to obtain results that are independent of species richness. The effectiveness of a typology system can be quantified by the ratio: <span class="math display">\[\Delta AUC\zeta = \frac{AUC\zeta^w}{AUC\zeta^b}\]</span> where values &gt; 1 indicate lower turnover within types than between types.</p>
</section>
<section id="deriving-benchmarks" class="level3">
<h3 class="anchored" data-anchor-id="deriving-benchmarks">Deriving benchmarks</h3>
</section>
<section id="software" class="level3">
<h3 class="anchored" data-anchor-id="software">Software</h3>
<p>To ensure replicability, we will use the groundhog R package () to use the packageversions that were available on the (add date). Posterior checks were conducted with coda … (), Data wrangling was done with data.table, and tidyverse packages.</p>
</section>
</section>
</section>
<section id="acknowledgements" class="level1">
<h1>Acknowledgements</h1>
</section>
<section id="references" class="level1 unnumbered">


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="list">
<div id="ref-alahuhtaComparativeAnalysisReveals2017" class="csl-entry" role="listitem">
Alahuhta J, Ecke F, Johnson LB, et al (2017) A comparative analysis reveals little evidence for niche conservatism in aquatic macrophytes among four areas on two continents. Oikos 126:136–148. <a href="https://doi.org/10.1111/oik.03154">https://doi.org/10.1111/oik.03154</a>
</div>
<div id="ref-amatulliHydrography90mNewHighresolution2022" class="csl-entry" role="listitem">
Amatulli G, Garcia Marquez J, Sethi T, et al (2022) Hydrography90m: A new high-resolution global hydrographic dataset. Earth System Science Data 14:4525–4550. <a href="https://doi.org/10.5194/essd-14-4525-2022">https://doi.org/10.5194/essd-14-4525-2022</a>
</div>
<div id="ref-amatulliGeomorpho90mEmpiricalEvaluation2020" class="csl-entry" role="listitem">
Amatulli G, McInerney D, Sethi T, et al (2020) Geomorpho90m, empirical evaluation and accuracy assessment of global high-resolution geomorphometric layers. Scientific Data 7:162. <a href="https://doi.org/10.1038/s41597-020-0479-6">https://doi.org/10.1038/s41597-020-0479-6</a>
</div>
<div id="ref-andersonNewMethodNonparametric2001" class="csl-entry" role="listitem">
Anderson MJ (2001) A new method for non-parametric multivariate analysis of variance. Austral Ecology 26:32–46. <a href="https://doi.org/10.1111/j.1442-9993.2001.01070.pp.x">https://doi.org/10.1111/j.1442-9993.2001.01070.pp.x</a>
</div>
<div id="ref-bergHydrologyRelatedClimate2021" class="csl-entry" role="listitem">
Berg P, Photiadou C, Bartosova A, et al (2021) <a href="https://doi.org/10.24381/cds.73237ad6">Hydrology related climate impact indicators from 1970 to 2100 derived from bias adjusted <span>European</span> climate projections</a>
</div>
<div id="ref-bevanRegionalBiomesOutperform2024" class="csl-entry" role="listitem">
Bevan PA, Ferreira GB, Ingram DJ, et al (2024) Regional <span>Biomes</span> outperform broader spatial units in capturing biodiversity responses to land-use change. Ecography e07318. <a href="https://doi.org/10.1111/ecog.07318">https://doi.org/10.1111/ecog.07318</a>
</div>
<div id="ref-blanchetCooccurrenceNotEvidence2020" class="csl-entry" role="listitem">
Blanchet FG, Cazelles K, Gravel D (2020) Co-occurrence is not evidence of ecological interactions. Ecology Letters 23:1050–1063. <a href="https://doi.org/10.1111/ele.13525">https://doi.org/10.1111/ele.13525</a>
</div>
<div id="ref-bogota-gregoryGeomorphologicalHabitatType2023" class="csl-entry" role="listitem">
Bogotá-Gregory JD, Jenkins DG, Lima FCT, et al (2023) Geomorphological habitat type drives variation in temporal species turnover but not temporal nestedness in <span>Amazonian</span> fish assemblages. Oikos e09967. <a href="https://doi.org/10.1111/oik.09967">https://doi.org/10.1111/oik.09967</a>
</div>
<div id="ref-brunCHELSABIOCLIMNovelSet2022" class="csl-entry" role="listitem">
Brun P, Zimmermann NE, Hari C, et al (2022) <span>CHELSA-BIOCLIM</span>+ <span>A</span> novel set of global climate-related predictors at kilometre-resolution
</div>
<div id="ref-burnerTraitsMediateNiches2021" class="csl-entry" role="listitem">
Burner RC, Stephan JG, Drag L, et al (2021) Traits mediate niches and co-occurrences of forest beetles in ways that differ among bioclimatic regions. Journal of Biogeography 48:3145–3157. <a href="https://doi.org/10.1111/jbi.14272">https://doi.org/10.1111/jbi.14272</a>
</div>
<div id="ref-clarkeNonparametricMultivariateAnalyses1993" class="csl-entry" role="listitem">
Clarke KR (1993) Non-parametric multivariate analyses of changes in community structure. Austral Ecology 18:117–143. <a href="https://doi.org/10.1111/j.1442-9993.1993.tb00438.x">https://doi.org/10.1111/j.1442-9993.1993.tb00438.x</a>
</div>
<div id="ref-cliffSpatialAutocorrelation1973" class="csl-entry" role="listitem">
Cliff AD, Ord JK (1973) Spatial autocorrelation. Pion, London
</div>
<div id="ref-rgiconsortiumRandolphGlacierInventory2017" class="csl-entry" role="listitem">
Consortium R (2017) Randolph <span>Glacier Inventory</span> – <span>A Dataset</span> of <span>Global Glacier Outlines</span>: <span>Version</span> 6.0: <span>Technical Report</span>, <span>Global Land Ice Measurements</span> from <span>Space</span>, <span>Colorado</span>, <span>USA</span>. <span>Digital Media</span>.
</div>
<div id="ref-cornuDistributionGroundwaterHabitats2013" class="csl-entry" role="listitem">
Cornu J-F, Eme D, Malard F (2013) The distribution of groundwater habitats in <span>Europe</span>. Hydrogeology Journal 21:949–960. <a href="https://doi.org/10.1007/s10040-013-0984-1">https://doi.org/10.1007/s10040-013-0984-1</a>
</div>
<div id="ref-crouchFalseEvidenceRate2025" class="csl-entry" role="listitem">
Crouch DJM (2025) The false evidence rate: <span>An</span> approach to frequentist error rate control conditioning on the observed <span><em>P</em></span> value. Proceedings of the National Academy of Sciences 122:e2415706122. <a href="https://doi.org/10.1073/pnas.2415706122">https://doi.org/10.1073/pnas.2415706122</a>
</div>
<div id="ref-decaceresImprovingIndicatorSpecies2010" class="csl-entry" role="listitem">
De Cáceres M, Legendre P, Moretti M (2010) Improving indicator species analysis by combining groups of sites. Oikos 119:1674–1684. <a href="https://doi.org/10.1111/j.1600-0706.2010.18334.x">https://doi.org/10.1111/j.1600-0706.2010.18334.x</a>
</div>
<div id="ref-denisonIntegratingRegionalFrameworks2021" class="csl-entry" role="listitem">
Denison CD, Scott MC, Kubach KM, Peoples BK (2021) Integrating <span>Regional Frameworks</span> and <span>Local Variability</span> for <span>Riverine Bioassessment</span>. Environmental Management 68:126–145. <a href="https://doi.org/10.1007/s00267-021-01479-6">https://doi.org/10.1007/s00267-021-01479-6</a>
</div>
<div id="ref-dormannBioticInteractionsSpecies2018" class="csl-entry" role="listitem">
Dormann CF, Bobrowski M, Dehling DM, et al (2018) Biotic interactions in species distribution modelling: 10 questions to guide interpretation and avoid false conclusions. Global Ecology and Biogeography 27:1004–1016. <a href="https://doi.org/10.1111/geb.12759">https://doi.org/10.1111/geb.12759</a>
</div>
<div id="ref-dufreneSpeciesAssemblagesIndicator1997" class="csl-entry" role="listitem">
Dufrêne M, Legendre P (1997) Species <span>Assemblages And Indicator Species</span>:the <span>Need For A Flexible Asymmetrical Approach</span>. Ecological Monographs 67:345–366. <a href="https://doi.org/10.1890/0012-9615(1997)067[0345:SAAIST]2.0.CO;2">https://doi.org/10.1890/0012-9615(1997)067[0345:SAAIST]2.0.CO;2</a>
</div>
<div id="ref-eeaEUHydroRiverNetwork2019" class="csl-entry" role="listitem">
EEA (2019) <a href="https://doi.org/10.2909/393359a7-7ebd-4a52-80ac-1a18d5f3db9c"><span>EU-Hydro</span> – <span>River Network Database</span></a>
</div>
<div id="ref-eeaPotentialFloodproneArea2020" class="csl-entry" role="listitem">
EEA (2020) Potential flood-prone area extent
</div>
<div id="ref-garcia-gironTimecalibratedTreeLife2024" class="csl-entry" role="listitem">
García-Girón J, Múrria C, Arnedo MA, et al (2024) A time-calibrated <span>“<span>Tree</span> of <span>Life</span>”</span> of aquatic insects for knitting historical patterns of evolution and measuring extant phylogenetic biodiversity across the world. Earth-Science Reviews 252:104767. <a href="https://doi.org/10.1016/j.earscirev.2024.104767">https://doi.org/10.1016/j.earscirev.2024.104767</a>
</div>
<div id="ref-huiZetaDiversityConcept2014" class="csl-entry" role="listitem">
Hui C, McGeoch MA (2014) Zeta <span>Diversity</span> as a <span>Concept</span> and <span>Metric That Unifies Incidence-Based Biodiversity Patterns</span>. The American Naturalist 184:684–694. <a href="https://doi.org/10.1086/678125">https://doi.org/10.1086/678125</a>
</div>
<div id="ref-jupkeEuropeanRiverTypologies2023" class="csl-entry" role="listitem">
Jupke JF, Birk S, Apostolou A, et al (2023) European river typologies fail to capture diatom, fish, and macrophyte community composition. Science of The Total Environment 896:165081. <a href="https://doi.org/10.1016/j.scitotenv.2023.165081">https://doi.org/10.1016/j.scitotenv.2023.165081</a>
</div>
<div id="ref-koppJointSpeciesDistribution2023" class="csl-entry" role="listitem">
Kopp D, Stoddard JL, Hill R, et al (2023) Joint species distribution models reveal taxon-specific sensitivities to potential anthropogenic alteration. Freshwater Science 726283. <a href="https://doi.org/10.1086/726283">https://doi.org/10.1086/726283</a>
</div>
<div id="ref-kunzTacklingInconsistenciesFreshwater2022" class="csl-entry" role="listitem">
Kunz S, Kefford BJ, Schmidt-Kloiber A, et al (2022) Tackling inconsistencies among freshwater invertebrate trait databases: Harmonising across continents and aggregating taxonomic resolution. Freshwater Biology 67:275–291. <a href="https://doi.org/10.1111/fwb.13840">https://doi.org/10.1111/fwb.13840</a>
</div>
<div id="ref-lecocqTOFFDatabaseTraits2019" class="csl-entry" role="listitem">
Lecocq T, Benard A, Pasquet A, et al (2019) <span>TOFF</span>, a database of traits of fish to promote advances in fish aquaculture. Scientific Data 6:301. <a href="https://doi.org/10.1038/s41597-019-0307-z">https://doi.org/10.1038/s41597-019-0307-z</a>
</div>
<div id="ref-linkeGlobalHydroenvironmentalSubbasin2019" class="csl-entry" role="listitem">
Linke S, Lehner B, Ouellet Dallaire C, et al (2019) Global hydro-environmental sub-basin and river reach characteristics at high spatial resolution. Scientific Data 6:283. <a href="https://doi.org/10.1038/s41597-019-0300-6">https://doi.org/10.1038/s41597-019-0300-6</a>
</div>
<div id="ref-maierJustifyYourAlpha2022" class="csl-entry" role="listitem">
Maier M, Lakens D (2022) Justify <span>Your Alpha</span>: <span>A Primer</span> on <span>Two Practical Approaches</span>. Advances in Methods and Practices in Psychological Science 5:1–14
</div>
<div id="ref-messagerEstimatingVolumeAge2016" class="csl-entry" role="listitem">
Messager ML, Lehner B, Grill G, et al (2016) Estimating the volume and age of water stored in global lakes using a geo-statistical approach. Nature Communications 7:13603. <a href="https://doi.org/10.1038/ncomms13603">https://doi.org/10.1038/ncomms13603</a>
</div>
<div id="ref-panagosGlobalRainfallErosivity2023" class="csl-entry" role="listitem">
Panagos P, Hengl T, Wheeler I, et al (2023) Global rainfall erosivity database (<span>GloREDa</span>) and monthly <span class="nocase">R-factor</span> data at 1&nbsp;km spatial resolution. Data in Brief 50:109482. <a href="https://doi.org/10.1016/j.dib.2023.109482">https://doi.org/10.1016/j.dib.2023.109482</a>
</div>
<div id="ref-panagosEuropeanSoilData2022" class="csl-entry" role="listitem">
Panagos P, Van Liedekerke M, Borrelli P, et al (2022) European <span>Soil Data Centre</span> 2.0: <span>Soil</span> data and knowledge in support of the <span>EU</span> policies. European Journal of Soil Science 73:e13315. <a href="https://doi.org/10.1111/ejss.13315">https://doi.org/10.1111/ejss.13315</a>
</div>
<div id="ref-pharaohPotentialDriversChanging2024" class="csl-entry" role="listitem">
Pharaoh E, Diamond M, Jarvie HP, et al (2024) Potential drivers of changing ecological conditions in <span>English</span> and <span>Welsh</span> rivers since 1990. Science of The Total Environment 946:174369. <a href="https://doi.org/10.1016/j.scitotenv.2024.174369">https://doi.org/10.1016/j.scitotenv.2024.174369</a>
</div>
<div id="ref-pinkertPotentialEcoregionalRange2023" class="csl-entry" role="listitem">
Pinkert S, Sica YV, Winner K, Jetz W (2023) The potential of ecoregional range maps for boosting taxonomic coverage in ecology and conservation. Ecography e06794. <a href="https://doi.org/10.1111/ecog.06794">https://doi.org/10.1111/ecog.06794</a>
</div>
<div id="ref-popovicFourPrinciplesImproved2024" class="csl-entry" role="listitem">
Popovic G, Mason TJ, Drobniak SM, et al (2024) Four principles for improved statistical ecology. Methods in Ecology and Evolution 2041–210X.14270. <a href="https://doi.org/10.1111/2041-210X.14270">https://doi.org/10.1111/2041-210X.14270</a>
</div>
<div id="ref-raboskyInverseLatitudinalGradient2018" class="csl-entry" role="listitem">
Rabosky DL, Chang J, Title PO, et al (2018) An inverse latitudinal gradient in speciation rate for marine fishes. Nature 559:392–395. <a href="https://doi.org/10.1038/s41586-018-0273-1">https://doi.org/10.1038/s41586-018-0273-1</a>
</div>
<div id="ref-robertsLabdsvOrdinationMultivariate2023" class="csl-entry" role="listitem">
Roberts DW (2023) Labdsv: <span>Ordination</span> and multivariate analysis for ecology
</div>
<div id="ref-szoszkiewiczMacrophyteIndexRivers2019" class="csl-entry" role="listitem">
Szoszkiewicz K, Jusik S, Pietruczuk K, Gebler D (2019) The <span>Macrophyte Index</span> for <span>Rivers</span> (<span>MIR</span>) as an <span>Advantageous Approach</span> to <span>Running Water Assessment</span> in <span>Local Geographical Conditions</span>. Water 12:108. <a href="https://doi.org/10.3390/w12010108">https://doi.org/10.3390/w12010108</a>
</div>
<div id="ref-tjurCoefficientsDeterminationLogistic2009" class="csl-entry" role="listitem">
Tjur T (2009) Coefficients of <span>Determination</span> in <span>Logistic Regression Models</span>—<span>A New Proposal</span>: <span>The Coefficient</span> of <span>Discrimination</span>. The American Statistician 63:366–372. <a href="https://doi.org/10.1198/tast.2009.08210">https://doi.org/10.1198/tast.2009.08210</a>
</div>
<div id="ref-tothNewGenerationHydraulic2015" class="csl-entry" role="listitem">
Tóth B, Weynants M, Nemes A, et al (2015) New generation of hydraulic pedotransfer functions for <span>Europe</span>. European Journal of Soil Science 66:226–238. <a href="https://doi.org/10.1111/ejss.12192">https://doi.org/10.1111/ejss.12192</a>
</div>
<div id="ref-valleSpeciesAssociationsJoint2023" class="csl-entry" role="listitem">
Vallé C, Poggiato G, Thuiller W, et al (2023) Species associations in joint species distribution models: From missing variables to conditional predictions. Journal of Biogeography jbi.14752. <a href="https://doi.org/10.1111/jbi.14752">https://doi.org/10.1111/jbi.14752</a>
</div>
<div id="ref-vansickleUsingMeanSimilarity1997" class="csl-entry" role="listitem">
Van Sickle J (1997) Using <span>Mean Similarity Dendrograms</span> to <span>Evaluate Classifications</span>. Journal of Agricultural, Biological, and Environmental Statistics 2:370–388. <a href="https://doi.org/10.2307/1400509">https://doi.org/10.2307/1400509</a>
</div>
<div id="ref-westfallResamplingbasedMultipleTesting1993" class="csl-entry" role="listitem">
Westfall P, Young S (1993) Resampling-based multiple testing: <span>Examples</span> and methods for p-<span>Value</span> adjustment. Wiley
</div>
<div id="ref-witjesEcodatacubeEuAnalysisready2023" class="csl-entry" role="listitem">
Witjes M, Parente L, Križan J, et al (2023) Ecodatacube.eu: Analysis-ready open environmental data cube for <span>Europe</span>. PeerJ 11:e15478. <a href="https://doi.org/10.7717/peerj.15478">https://doi.org/10.7717/peerj.15478</a>
</div>
<div id="ref-wulffHowWhyAlpha2023" class="csl-entry" role="listitem">
Wulff JN, Taylor L (2023) How and why alpha should depend on sample size: <span class="nocase">A Bayesian-frequentist</span> compromise for significance testing. PsyArXiv
</div>
<div id="ref-yangWhyFunctionalTraits2018" class="csl-entry" role="listitem">
Yang J, Cao M, Swenson NG (2018) Why <span>Functional Traits Do Not Predict Tree Demographic Rates</span>. Trends in Ecology &amp; Evolution 33:326–336. <a href="https://doi.org/10.1016/j.tree.2018.03.003">https://doi.org/10.1016/j.tree.2018.03.003</a>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>